toast-tk-rest-utils/src/main/java/io/toast/tk/core/rest/HttpRequest.java

```diff
@@ -0,0 +1,37 @@
+package io.toast.tk.core.rest;
+
+public class HttpRequest {
+
+	public final String uri;
+	public String apiKey;
+	public String json;
+	public String proxyAdress;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make proxyAdress a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)